packages/dx-react-scheduler-material-ui/src/templates/common/dialog/overlay.jsx

```diff
@@ -1,25 +1,33 @@
 import * as React from 'react';
+import { styled } from '@mui/material/styles';
 import * as PropTypes from 'prop-types';
 import classNames from 'clsx';
 import Dialog from '@mui/material/Dialog';
-import withStyles from '@mui/styles/withStyles';
 
-const styles = {
-  modal: {
+const PREFIX = 'Overlay';
+
+export const classes = {
+  modal: `${PREFIX}-modal`,
+  paper: `${PREFIX}-paper`,
+  root: `${PREFIX}-root`,
+};
+
+const StyledDialog = styled(Dialog)({
+  [`&.${classes.modal}`]: {
     position: 'absolute!important',
   },
-  paper: {
+  [`& .${classes.paper}`]: {
```

Comment:
Class should be without spacing - it is passed to the dialog props

Reply:
Classes in props is better not take into consideration , since the width of the tooltip has already broken